project.clj

```diff
@@ -1,9 +1,17 @@
+(def clj-version "1.7.0")
+(def ks-version "1.3.0")
+(def tk-version "1.3.1")
+
 (defproject puppetlabs/ring-middleware "0.2.3-SNAPSHOT"
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/tools.logging "0.2.6"]
-                 [ring "1.3.0" :exclusions [org.clojure/java.classpath]]
-                 [puppetlabs/ssl-utils "0.8.0" :exclusions [clj-time]]
-                 [puppetlabs/http-client "0.4.4" :exclusions [prismatic/schema commons-io clj-time]]]
+  :dependencies [[cheshire "5.6.1"]
+                 [org.clojure/clojure ~clj-version]
+                 [org.clojure/tools.logging "0.3.1"]
+                 [prismatic/schema "1.1.0"]
+                 [puppetlabs/http-client "0.5.0" :exclusions [prismatic/schema commons-io clj-time ]]
+                 [puppetlabs/kitchensink ~ks-version :exclusions [clj-time]]
+                 [puppetlabs/ssl-utils "0.8.1" :exclusions [clj-time]]
+                 [ring "1.4.0" :exclusions [org.clojure/java.classpath]]
+                 [slingshot "0.12.2"]]
```

Comment:
We generally avoid using `:exclusions`.  [Explanation here](https://confluence.puppetlabs.com/pages/viewpage.action?pageId=48956980)  Instead we have two sections of dependencies here - one for the direct dependencies and another which just resolves version conflicts in transitive dependencies.